conftest.py

```
@@ -460,10 +459,15 @@ def environment_cleanup(monkeypatch: pytest.MonkeyPatch) -> None:
 
 @pytest.fixture(scope="function", autouse=True)
 def check_output_dir() -> None:
-    global test_output_dirs
-    assert os.path.isdir(test_output_root)
-    assert len(os.listdir(test_output_root)) >= test_output_dirs
-    test_output_dirs = len(os.listdir(test_output_root))
+    try:
+        global test_output_dirs
+        assert os.path.isdir(test_output_root)
+        assert len(os.listdir(test_output_root)) >= test_output_dirs
+        test_output_dirs = len(os.listdir(test_output_root))
+    except Exception:
```

As this fixture is intended to prevent from running tests without output dirs, doesn't swallowing the exception make this fixture useless?

---

removed